tests/shell/shell_recorder/script.dat

```diff
@@ -0,0 +1,13 @@
+Storage: dump
+Mountpoint: system/testmount/test
+File: /tmp/test.dump
+MountArgs:
```

Comment:
Storage is the storage plugin, i was planing on renaming it later . 
MountArgs is just a String that gets attached after the storage plugin name, like additional plugins, plugin configs, etc
All those variables are optional.